server/liveblog/themes/themes_assets/classic/views/posts.html

```diff
@@ -4,13 +4,52 @@
          ng-class="['lb-mainitem-{{ post.mainItem.item_type | varname }}', {'show-author-avatar': 
                         ctrl.timeline.settings.showAuthor &&
                         ctrl.timeline.settings.showAuthorAvatar &&
-                        ctrl.isAd(post),
+                        !ctrl.isAd(post),
                     'lb-post-permalink-selected': post._id === timeline.permalink._id}]">
-    <span ng-if="ctrl.isAd(post)">
+    <span ng-if="!ctrl.isAd(post)">
         <lb-author item="post.mainItem" timeline="timeline"></lb-author>
         <div class="lb-post-highlighted" ng-if="post.lb_highlight">
             <i class="icon-star icon--orange"></i>
         </div>
+
+        <div class="lb-post-permalink" style="right:30px" ng-click="showSocial = !showSocial">
```

Comment:
can we have an `ng-if` here with some theme setting ex:
`ctrl.timeline.settings.showSocialShare`